.github/PULL_REQUEST_TEMPLATE/develop.md

```diff
@@ -0,0 +1,33 @@
+# Pull Request Details
+<!--- The title of the PR should summarize the change implemented. -->
+
+## Description
+<!--- Provide a general summary of your changes in the title above -->
+<!--- Why is this change required? What problem does it solve? -->
+
+## Related Issue
+<!--- Refer to any related issues here -->
+<!--- If this PR fully addresses an issue, please say "Fixes #1234", -->
```

Comment:
Are we using github issues or trello cards?

Reply:
I think we were doing bugs on GH, but trello for work items.